### PR TITLE
Revert declaring cover_layout field in content type schema as readonly

### LINKED ISSUE #761: TypeError: Can't set values on read-only fields (name=cover_layout, class=collective.cover.content.Cover)
Seems the workaround introduced in #700 was in fact a bad idea and is causing other issues like the one descibed in https://github.com/plonegovbr/brasil.gov.portal/issues/388

We need to find another way to fix this, probably the one described in https://github.com/plone/plone.app.testing/issues/39

### PR DESCRIPTION
**Pull request by hvelarde**
fixes #761 
refs: https://github.com/plone/plone.app.testing/issues/39

### CODE REVIEW

File: src/collective/cover/testing.py [hunk 1]
```diff
@@ -115,6 +115,16 @@ def generate_jpeg(width, height):
...
 
 
+# FIXME: workaround for https://github.com/plone/plone.app.testing/issues/39
+#        Products.TinyMCE is used only in Plone 4
+if not IS_PLONE_5:
+    autoform = ('plone.autoform', {'loadZCML': True})
+    tinymce = ('Products.TinyMCE', {'loadZCML': True})
+    products = list(PLONE_FIXTURE.products)
+    products.insert(products.index(tinymce), autoform)
+    PLONE_FIXTURE.products = tuple(products)
```
Comment (idgserpro) [inline src/collective/cover/testing.py]: We have to do this in ```brasil.gov.portal``` if we want to update Plone.

But honestly I think this should be fixed in plone.app.testing.

Comment (hvelarde) [reply src/collective/cover/testing.py]: I need to keep going on with development; we have deadlines.

**Comment by idgserpro** [comment]
Then put it on ```brasil.gov.portal```.

File: src/collective/cover/testing.py (hunk 1 above)
Comment (idgserpro) [reply src/collective/cover/testing.py]: Then put it on ```brasil.gov.portal```.

Comment (hvelarde) [reply src/collective/cover/testing.py]: done in d530a062cbc8b9fc71c0d38fcd1c2ed23da32825

**Pull request merged**